tools/qcxms/qcxms_prod_run.xml

```diff
@@ -0,0 +1,90 @@
+<tool id="qcxms_prod_run" name="QCxMS prod run" version="@TOOL_VERSION@+galaxy0" python_template_version="3.5" profile="21.05">
+    <description>production runs</description>
+    
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+
+    <expand macro="creator"/>
+
+    <edam_topics>
+        <edam_topic>topic_3332</edam_topic>
+    </edam_topics>
+    <edam_operations>
+        <edam_operation>operation_0297</edam_operation>
+    </edam_operations>
```

Comment:
Do we have a bio.tools entry for the package?

Reply:
no